src/desktop/apps/artwork/components/commercial/templates/price.jade

```diff
@@ -8,11 +8,9 @@ if artwork.sale_message || isPermanentCollection
         .artwork-commercial__price-label Price
         .artwork-commerical__price
           | #{artwork.sale_message}
-          span(
-            class= artwork.is_price_range ? 'help-tooltip' : ''
-            data-message= 'The range is an approximate indication of the work’s price point, and the exact price is quoted upon request.'
-            data-anchor= 'bottom-right'
-          )
+          if artwork.is_price_range
+            .react-tooltip-container
+              != stitch.components.Tooltip({message: 'The range is an approximate indication of the work’s price point, and the exact price is quoted upon request.', hoverAlign: 'left' })
```

Comment:
I would try this with `yarn watch` running against a local version of reaction and see how it goes

Reply:
Since we can't do multiline in jade might be good to move the `hoverAlign: 'left'` prop to the front so its not lost to future devs behind the long `message` string.